.github/workflows/reusable-e2e.yml

```diff
@@ -327,7 +334,7 @@ jobs:
 
   e2e-ios:
     needs: check-changed-packages
-    if: ${{ needs.check-changed-packages.outputs.has-changed-packages == 'true' }}
+    if: ${{ inputs.skip-changed-packages-check || needs.check-changed-packages.outputs.has-changed-packages == 'true' }}
```

Comment:
Not going to pretend i understand how the syntax works here but is it possible to align the syntax of the two conditions?

Reply:
We cannot align the two. The latter - `needs.check-changed-packages.outputs.has-changed-packages`  is the result of the check that gets skipped.

Reply:
Think we can put 
```suggestion
    if: ${{ inputs.skip-changed-packages-check == 'true' || needs.check-changed-packages.outputs.has-changed-packages == 'true' }}
```

Reply:
Is this valid? Thinking it make sense that output is a string. But let me kick off a run to test it out

Reply:
oh wait, yeah this might not work as expected because you have the input type as `input` `type` as a `boolean`.

We've used `string` because of this issue https://github.com/actions/runner/issues/1483 but looks like it's fixed since then.

Reply:
Up to thoughts, but what do you think about enforcing the input type to be a `string` for the sake of consistency? We can look at the new `boolean` inputs and change our workflows holistically.

Reply:
That's a good history reference. No wonder our conditionals try to evaluate a string

Reply:
I am good to make it consistent for now and revisit the workflows as a whole

Reply:
Updated @wlee221 @calebpollman